src/core/server/core_usage_data/types.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface CoreUsageData {
+  config: CoreConfigUsageData;
+  // services: CoreServicesUsageData;
```

Comment:
Because the telemetry team asked us not to use `...Telemetry` I've switched to `...UsageData` but `CoreUsageData['usageData']` is a bit awkward, so I deviated from the "spec" in the original issue by using the key "services" with the idea that we would group usage data under the respective Core services they original from e.g. savedObjects / plugins.